### ISSUE TITLE
Contrast of the input labels are not adequate

### ISSUE THREAD

**Issue by LieutenantRoger** [topcoder-platform/micro-frontends-challenges-app#222]
**Summary:**
The contrast of the textbox labels do not comply with thw WCAG standards

**Steps to Reproduce:**

1. Open https://platform.topcoder.com/earn/find/challenges
2. Observe any input boxes labels

**Actual Result:**
The contrast of the label with the background is less then the minimum standard

**Expected Result:**
The contrast should be higher

**Environment Details:**

***Devices(s):***
MacBook Pro
***Operating System and version:***
MacOS Catalina
***Browser and version:***
Safari 14.0.3
**Screen recording or Screen snapshot:**
![Screenshot_2021-10-29_at_17.14.30](/uploads/c045502e97df5c9c56ab02490e11e901/Screenshot_2021-10-29_at_17.14.30.png)
